src/WinRT.Runtime/ComWrappersSupport.cs

```diff
@@ -181,7 +181,8 @@ internal static List<ComInterfaceEntry> GetInterfaceTableEntries(Type type)
             // which means the attribute lives on the authoring metadata type.
             if (winrtExposedClassAttribute == null)
             {
-                var authoringMetadaType = type.GetRuntimeClassCCWType();
+                // Using GetCCWType rather than GetRuntimeClassCCWType given we want to handle boxed value types.

```

Comment:
Should we think about this will play with cases where `IReference<T>` support is disabled? Would thast just fail elsewhere when we actually try to add the `IReference<T>` support, or should we add a check here for that feature flag and throw right away if the type is a value type, or something else?

Reply:
I think when we check the attribute, that is already going to return an empty list for structs and delegates.  So, I don't think we need a check here.  And we want to recognize here that there was indeed an attribute and not go on the fallback code path.